src/Libraries/CoreNodes/FileSystem.cs

```diff
@@ -141,7 +146,8 @@ public static void AppendText(string filePath, string text)
         /// <summary>
         ///     Combines multiple strings into a single file path.
         /// </summary>
-        /// <param name="paths">String to combine into a path.</param>
+        /// <param name="paths">String to combine into a path</param>
```

Comment:
This should say `Strings` as the more common use case is multiple string values.

Reply:
Adressed in b7a80b0